.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,27 @@
+name: Prepare Release From main
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'release version, e.g. 0.14.0'
+        required: true
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Setup node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Run script
+        run: |
+          git config --global user.name 'Ute Weiss'
+          git config --global user.email 'ute.weiss@tngtech.com'
+          VERSION=${{ github.event.inputs.version }}
+          ./prepare_release.sh --version=$VERSION --upstream=https://github.com/${{ github.repository }}
```

Comment:
What would happen here if this workflow runs on a fork?

Reply:
I guess that fork would have a release branch with a commit by me lying around then? Maybe we change the name and email to something saying "technical user" ...

Reply:
for the username we probably can use github.actor. I guess for a manually triggered workflow that should be set - for email some fake no-reply address?

Reply:
changed it to github.actor and the actors github noreply mail adress (@users.noreply.github.com)
also for the update_notice workflow - the github.actor is the user who triggered the workflow run or whose commit triggered the run, apparently (I tried it - github.actor is also set if a push event triffered the worflow run)